bin/cron/create_rollup_tables

```diff
@@ -15,6 +15,28 @@ def sanitize_string_for_db(unsanitized)
   unsanitized.gsub(/[\r\n]+/, '').gsub(/"/, '\"')
 end
 
+# @param contained_level [Level] The level to parse.
+# @return [String] The level question, from the markdown or the text. If both
+#   exist, the two are concatenated.
+def get_contained_question(contained_level)
+  contained_question_markdown = contained_level.properties['markdown']
+
+  contained_question_text = nil
+  if contained_level.properties['questions']
+    contained_question_text = contained_level.
+      properties['questions'].
+      first['text']
+  end
+
+  if contained_question_markdown.nil?
+    return contained_question_text
+  end
+  if contained_question_text.nil?
+    return contained_question_markdown
+  end
```

Comment:
might we want `if contained_question_markdown.nil? || contained_question_markdown.empty?`?